credentials.md

```diff
@@ -20,27 +20,34 @@ Example: `/ocpi/cpo/2.0/credentials` and `/ocpi/emsp/2.0/credentials`
 
 ### 1.1 __GET__ Method
 
-Retrieves the client's credentials for the server's platform. The request body is empty, the response contains the credentials for the server's platform.
+Retrieves the credentials object to access the server's platform. The request body is empty, the response contains the credentials object to access the server's platform. This credentials object also contains extra information about the server such as its business details.
 
 
 ### 1.2 __POST__ Method
 
-Provides the server with credentials to the client's system, this initiates the registration process for this endpoint's version. The server must also fetch the client's endpoints for this version. 
+Provides the server with credentials to access the client's system. This credentials object also contains extra information about the client such as its business details.
 
-If successful, the server must generate a new token and respond with the client's new credentials to the server's system.
+A `POST` initiates the registration process for this endpoint's version. The server must also fetch the client's endpoints for this version.
 
+If successful, the server must generate a new token and respond with the client's new credentials to access the server's system. The credentials object in the response also contains extra information about the server such as its business details.
+
+This must return a `405 method not allowed` if the client was already registered.
 
 ### 1.3 __PUT__ Method
 
-Updates the server's credentials to the client's system and switches to the version that contains this credentials endpoint. The server must also fetch the client's endpoints again (for this version) as they could have been updated.
+Provides the server with updated credentials to access the client's system, This credentials object also contains extra information about the client such as its business details.
+
+A `PUT` will switch to the version that contains this credentials endpoint if it's different from the current version. The server must also fetch the client's endpoints again.
 
-If successful, the server must generate a new token for the client and respond with the client's updated credentials to the server's system.
+If successful, the server must generate a new token for the client and respond with the client's updated credentials to the server's system. The credentials object in the response also contains extra information about the server such as its business details.
```

Comment:
`the client's updated credentials to the server's system.` should be `the client's updated credentials to access the server's system.`